backend/apps/owasp/models/event.py

```diff
@@ -40,21 +41,57 @@ class Category(models.TextChoices):
         OTHER = "other", "Other"
         PARTNER = "partner", "Partner"
 
+    class Type(models.TextChoices):
+        """Event type."""
+
+        CALENDAR = "calendar", "Calendar"
+        EXISTING = "existing", "Existing"
+
+    class Status(models.TextChoices):
+        """Event status."""
+
+        CONFIRMED = "confirmed", "Confirmed"
+        TENTATIVE = "tentative", "Tentative"
+        CANCELLED = "cancelled", "Cancelled"
+
     category = models.CharField(
         verbose_name="Category",
         max_length=11,
         choices=Category.choices,
         default=Category.OTHER,
     )
+    calendar_id = models.CharField(
+        verbose_name="Calendar ID", max_length=1024, blank=True, default=""
+    )
+    channel_id = models.CharField(verbose_name="Channel ID", max_length=15, default="")
+    member = models.ForeignKey(
+        "slack.Member",
+        on_delete=models.CASCADE,
+        related_name="calendar_events",
+        verbose_name="Slack Member",
+        null=True,
+    )
```

Comment:
This doesn't look right in long term perspective. Let's decouple the notification/reminder part from event.

[This PR](https://github.com/OWASP/Nest/pull/2049/) is going to introduce entity channel model which can be used for event notifications too. We would probably need an extra model for `Reminder` set up.